bdb/bdb_api.h

```diff
@@ -2185,11 +2186,11 @@ int bdb_genid_set_format(bdb_state_type *bdb_state, int format);
 int bdb_genid_allow_original_format(bdb_state_type *bdb_state);
 int genid_contains_time(bdb_state_type *bdb_state);
 
-int bdb_llmeta_get_lua_sfuncs(char ***, int *, int *bdberr);
-int bdb_llmeta_add_lua_sfunc(char *, int *bdberr);
+int bdb_llmeta_get_lua_sfuncs(void *sfuncs, int *bdberr);
```

Comment:
Maybe declare a LISTC_T(struct function_name) type?  I know there's quite a few places where we pass void pointers between db and bdb, but no real good reason for this anymore.

Reply:
Good Idea. I think I'll do it in a separate PR along with mending the state diagram to add the `deterministic` keyword.